ui/apps/platform/src/Containers/NetworkGraph/NetworkGraph.tsx

```diff
@@ -105,7 +105,7 @@ const TopologyComponent = ({ model }: TopologyComponentProps) => {
 
     function onSelect(ids: string[]) {
         const newSelectedId = ids?.[0] || '';
-        const newSelectedEntity = getNodeById(model, newSelectedId);
+        const newSelectedEntity = getNodeById(model?.nodes, newSelectedId);
```

Comment:
shouldn't it just not call the function if there are no nodes?

Reply:
It's more for typescript. Seems like `nodes` is actually optional